### ISSUE TITLE
Admiral Akbar Rotation

### ISSUE THREAD

**Issue by miklevin** [cosmograph-org/cosmos#127]
Since you liked the Star Wars image, the feature request is named for it.

Scenario: You're Admiral Akbar using Cosmograph.app to show the Rebel Forces the critical air duct flaw. But each time you pull up the Death Star, the air duct appears to be in a different location due to the non-deterministic force graph rendering. I guess that helps compress hologram data.

Anyhoo, I have no illusions of real 3D rotation being possible. But I would love an 80/20-Rule solution as simple as controlling the original random number. I understand that the nondeterministic nature may result in different outcomes, but good enough may be good enough.

The other and more desirable request is to actually have a rotation feature. I can imagine it's a calculation nightmare, but also maybe it's a lovely challenge. Add a rotation slider. Maybe it only works when paused, I don't know. But what I do know is that the visual effect of rotation happens all the time as the force graph begins rendering. Sometimes the shape looks like it spins round and round before it settles down. Whatever is happening with that spinning is what I want that slider to do.

In other words, we need to be able to "pose" the shape. It's actually a desirable feature for the actual tongue-in-cheek Acmiral Akbar reason I state. If you're showing someone something important about their link graph, changing orientation about the important things you're looking at in each meeting ("seeing" your air duct in a different location) is too much for humans to process. Better for the machine to do it.
